lib/actions/preload.coffee

```diff
@@ -0,0 +1,206 @@
+###
+Copyright 2016-2017 Resin.io
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+###
+
+_ = require('lodash')
+form = require('resin-cli-form')
+preload = require('resin-preload')
+Promise = require('bluebird')
+resin = require('resin-sdk-preconfigured')
+
+dockerUtils = require('../utils/docker')
```

Comment:
In the CLI we don't do top-level requires in actions, as a rule. To keep startup time down, wherever possible each action requires its dependencies inside the action itself, so we only end up loading the dependencies we definitely need. Otherwise every command loads every module that every other command might want to use, which really hurts UX.

`resin deploy` is a good example: https://github.com/resin-io/resin-cli/blob/master/lib/actions/deploy.coffee#L146-L152